clickhouse/setup.py

```diff
@@ -12,6 +12,7 @@
     url="https://github.com/testcontainers/testcontainers-python",
     install_requires=[
         "testcontainers-core",
+        "clickhouse-connect",
```

Comment:
Given the updated requirements, let's update the lock files. Could you run `make requirements` from the root directory and push the changes?

Reply:
I see there have been some changes to requirements management. Is there anything I should do?

Reply:
You can run `python get_requirements.py --pr=[your PR number]` to update the requirements. Full details [here](https://github.com/testcontainers/testcontainers-python/blob/main/.github/PULL_REQUEST_TEMPLATE/new_container.md).